pkg/scc/scc.go

```diff
@@ -184,24 +268,94 @@ func (builder *Builder) WithAllowCapabilities(allowCapabilities []coreV1.Capabil
 	return builder
 }
 
+// WithDefaultAddCapabilities adds list of defaultAddCapabilities to SecurityContextConstraints.
+func (builder *Builder) WithDefaultAddCapabilities(defaultAddCapabilities []coreV1.Capability) *Builder {
+	if valid, _ := builder.validate(); !valid {
+		return builder
+	}
+
+	glog.V(100).Infof("Redefining SecurityContextConstraints %s with"+
+		"defaultAddCapabilities: %v", builder.Definition.Name, defaultAddCapabilities)
+
+	if builder.Definition.DefaultAddCapabilities == nil {
+		builder.Definition.DefaultAddCapabilities = defaultAddCapabilities
+
+		return builder
+	}
+
+	builder.Definition.DefaultAddCapabilities = append(builder.Definition.DefaultAddCapabilities,
+		defaultAddCapabilities...)
+
+	return builder
+}
+
+// WithPriority adds priority to SecurityContextConstraints.
+func (builder *Builder) WithPriority(priority *int32) *Builder {
+	if valid, _ := builder.validate(); !valid {
+		return builder
+	}
+
+	glog.V(100).Infof("Redefining SecurityContextConstraints %s with"+
+		"priority: %v", builder.Definition.Name, priority)
+
+	builder.Definition.Priority = priority
+
+	return builder
+}
+
 // WithFSGroup adds fsGroup to SecurityContextConstraints.
-func (builder *Builder) WithFSGroup(fsGroup string) *Builder {
+func (builder *Builder) WithFSGroup(fsGroupType string, fsGroupRange map[string]int64) *Builder {
```

Comment:
You are going to change public api functions WithFSGroup. This function is in use in few test cases. Let's avoid changing public api functions. 
Instead it's better to add additional WithFSGroupRange functions.

Reply:
I don't have a problem to change it back, just take in mind that parameter used as fsGroup is fsGroupType, so there is some inconsistency.

Reply:
I suggested to add new function. Please check below:

```
func (builder *Builder) WithFSGroupRange(fsGroupMin, fsGroupMax int64) *Builder {
	if valid, _ := builder.validate(); !valid {
		return builder
	}

	glog.V(100).Infof("Redefining SecurityContextConstraints %s with"+
		"fsGroupRange: fsGroupMin: %d, fsGroupMax: %d", builder.Definition.Name, fsGroupMin, fsGroupMax)

	if fsGroupMin > fsGroupMax {
		glog.V(100).Infof("SecurityContextConstraints 'fsGroupMin' argument can not be greater than fsGroupMax")

		builder.errorMsg = "SecurityContextConstraints 'fsGroupMin' argument can not be greater than fsGroupMax"

		return builder
	}

	if builder.Definition.FSGroup.Ranges == nil {
		builder.Definition.FSGroup.Ranges = []securityV1.IDRange{{Min: fsGroupMin, Max: fsGroupMax}}

		return builder
	}

	builder.Definition.FSGroup.Ranges = append(
		builder.Definition.FSGroup.Ranges, securityV1.IDRange{Max: fsGroupMax, Min: fsGroupMin})

	return builder
}
```

Reply:
Please revert WithFSGroup to it's original state.